docs/book/src/topics/machinepools.md

```diff
@@ -25,7 +25,7 @@ clusterctl config cluster my-cluster --kubernetes-version v1.16.8 --flavor machi
 ```
 
 The template used for this [flavor](https://cluster-api.sigs.k8s.io/clusterctl/commands/config-cluster.html#flavors)
-is located [here](../../templates/cluster-template-machinepool.yaml).
+is located [here](../../../../templates/cluster-template-machinepool.yaml).
```

Comment:
```suggestion
is located [here](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/templates/cluster-template-machinepool.yaml).
```
Only occurred to me now, but given that this page is published at https://cluster-api-aws.sigs.k8s.io/templates/cluster-template-machinepool.yaml , should be a full URL back to Github.